app/templates/activity.html

```diff
@@ -11,6 +11,23 @@
 <button type="submit">{{ gettext('Post') }}</button>
 </form>
 
+<h3>My Profile Progress</h3>
+
+<ul>
+  {% set q_progress = user.questionnaire_progress %}
+  {% for questionnaire in QUESTIONNAIRES %}
+    {% if q_progress[questionnaire.id].answered > 0 %}
+      <li>{{ questionnaire.name }}
+	<progress value="{{ q_progress[questionnaire.id].answered }}"
+		  max="{{ q_progress[questionnaire.id].total }}"></progress>
+      </li>
+    {% else %}
+      <li class="text-muted">{{ questionnaire.name }}</li>
+    {% endif %}
```

Comment:
Minor thing that would be nice to have here so styling is easier -- the logic to put a checkmark (can just use unicode checkmark ✓ for now) when the questionnaire is 100% done.